package.json

```diff
@@ -5,8 +5,7 @@
   "main": "index.html",
   "type": "module",
   "scripts": {
-    "format": "npx prettier --write .",
-    "lint": "npx eslint --fix .",
+    "lint": "npx prettier --loglevel silent --write . && npx eslint --quiet --fix .",
```

Comment:
This way you don't have to run two functions and the output will only be errors.